packages/strapi-design-system/src/SubNav/SubNavHeader.js

```diff
@@ -0,0 +1,140 @@
+
+const Searchbar = styled(FieldInput)`
+  height: ${32 / 16}rem;
+`;
+const CloseIconWrapper = styled(Row)`
+  font-size: 0.5rem;
+  svg path {
+    fill: ${({ theme }) => theme.colors.neutral400};
+  }
+`;
+const SearchIconWrapper = styled(Row)`
+  svg {
+    width: ${11 / 16}rem;
+    height: ${11 / 16}rem;
+    path {
+      fill: ${({ theme, focused }) => (focused ? theme.colors.primary600 : theme.colors.neutral800)};
+    }
+  }
+`;
+const CustomDivider = styled(Divider)`
+  width: ${24 / 16}rem;
+`;
+
+export const SubNavHeader = ({ as, label, searchLabel, searchable, onChange, value, onClear }) => {
+  const [isSearchOpen, setSearchOpen] = useState(false);
+  const previousSearchOpenValue = usePrevious(isSearchOpen);
+  const clearButonId = useId('subnav-searchbar-clear');
+  const searchRef = useRef();
+  const searchButtonRef = useRef();
+  const isCompleting = value.length > 0;
+
+  useEffect(() => {
+    if (isSearchOpen && searchRef.current) {
+      searchRef.current.focus();
+    }
+    if (previousSearchOpenValue && !isSearchOpen && searchButtonRef.current) {
+      searchButtonRef.current.focus();
+    }
+  }, [isSearchOpen]);
+
+  const toggleSearch = () => {
+    setSearchOpen((isOpen) => !isOpen);
+  };
+
+  const handleClear = (e) => {
+    onClear(e);
+    searchRef.current.focus();
+  };
+
+  const handleBlur = (e) => {
+    if (e.relatedTarget?.id !== clearButonId) {
+      // setSearchOpen(false);
+    }
+  };
+
+  const handleKeyDown = (e) => {
+    if (e.key === KeyboardKeys.ESCAPE) {
+      setSearchOpen(false);
+    }
+  };
+
+  if (isSearchOpen) {
+    return (
+      <Box paddingLeft={4} paddingTop={5} paddingBottom={2} paddingRight={4}>
+        <Searchbar
+          onKeyDown={handleKeyDown}
+          ref={searchRef}
+          onBlur={handleBlur}
+          value={value}
+          onChange={onChange}
+          startAction={
+            <SearchIconWrapper focused={isSearchOpen}>
+              <SearchIcon aria-hidden={true} />
+            </SearchIconWrapper>
+          }
+          endAction={
+            isCompleting ? (
+              <FieldAction id={clearButonId} label="Clear label" onClick={handleClear}>
+                <CloseIconWrapper>
+                  <CloseAlertIcon />
+                </CloseIconWrapper>
+              </FieldAction>
+            ) : undefined
+          }
+        />
+        <Box paddingLeft={2} paddingTop={4}>
+          <CustomDivider />
+        </Box>
+      </Box>
+    );
+  }
+
+  return (
+    <Box paddingLeft={6} paddingTop={6} paddingBottom={2} paddingRight={4}>
+      <Row justifyContent="space-between">
+        <H2 as={as}>{label}</H2>
+        {searchable && (
+          <IconButton ref={searchButtonRef} onClick={toggleSearch} label={searchLabel} icon={<Search />} />
+        )}
+      </Row>
+      <Box paddingTop={4}>
+        <CustomDivider />
+      </Box>
+    </Box>
+  );
+};
+
+SubNavHeader.defaultProps = {
+  as: 'h1',
```

Comment:
I would default to H2. H1 is supposed to be in the "main" or "header" of a page :)